freqtrade/freqai/freqai_interface.py

```diff
@@ -658,7 +660,7 @@ def fit_live_predictions(self, dk: FreqaiDataKitchen, pair: str) -> None:
 
         return
 
-    def inference_timer(self, do='start'):
+    def inference_timer(self, do: str = 'start', pair: str = ''):
```

Comment:
you might wanna type "do" as `Literal['start', 'stop']` ... that way mypy will ensure nobody calls this with something else.

Reply:
Done!